### ISSUE TITLE
Having a svg file inside /pages causes an error

### ISSUE THREAD

**Issue by tleunen** [gatsbyjs/gatsby#206]
Having a svg file (even never used) inside the pages directory causes an error in development.

I was able to reproduce it with the starter template.
```
WARNING in ./pages ^\.\/.*$
Module not found: Error: Cannot resolve module 'file-loader' in /Users/tleunen/dev/gatsby-default/def/pages
 @ ./pages ^\.\/.*$
```

**Comment by KyleAMathews** [comment]
So as it turns out, file-loader wasn't a gatsby dependency :-) Added that and published 0.8.11.

**Comment by KyleAMathews** [comment]
Thanks for the bug report!
